client/state/themes/reducer.js

```diff
@@ -301,6 +303,14 @@ export const queries = ( () => {
 		};
 	}
 
+	function fromApi( theme ) {
```

Comment:
Handling this in the reducer because this API request does not use the data layer.